modules/tensor_mechanics/src/materials/ADHillElastoPlasticityStressUpdate.C

```diff
@@ -54,7 +54,7 @@ ADHillElastoPlasticityStressUpdate::ADHillElastoPlasticityStressUpdate(
     _yield_condition(1.0),
     _yield_stress(getParam<Real>("yield_stress")),
     _hill_constants(getMaterialPropertyByName<std::vector<Real>>(_base_name + "hill_constants")),
-    _hill_tensor(6, 6)
+    _hill_tensor(getMaterialPropertyByName<DenseMatrix<Real>>(_base_name + "hill_tensor"))
```

Comment:
Just to clarify, there is no "use transformation: option here? The full tensor is always required?

Reply:
I'm asking because not always  declaring the full tensor property would be a way to make MOOSE error out if inconsistent options are used.

Reply:
The `hill_tensor` property is now always handled as a pointer to enable conditional declaration/retrieval.